src/Tools/ExternalAccess/MonoDevelop/Microsoft.CodeAnalysis.ExternalAccess.MonoDevelop.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.MonoDevelop</RootNamespace>
+    <TargetFramework>net472</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.MonoDevelop</PackageId>
+    <PackageDescription>
+      A supporting package for MonoDevelop:
+      https://github.com/mono/monodevelop
+    </PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--
+      ⚠ ONLY MONODEVELOP ASSEMBLIES MAY BE ADDED HERE ⚠
+    -->
+    <InternalsVisibleTo Include="MonoDevelop.Ide" Key="$(MonoDevelopKey)" />
```

Comment:
Xamarin stuff should be here too?

Reply:
Unless I'm misunderstanding the goal of this assembly, Workspaces.Desktop had a IVT to Xamarin.Ide:
https://github.com/dotnet/roslyn/blob/13a786a28873c39659bbaf2c23ca90369448cec4/src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj#L283

Reply:
These could be added if Xamarin.Ide and MonoDevelop are shared code bases. If the overlap is minimal or they are developed/distributed largely independently, it would probably make more sense to add a new adapter for Xamarin. Either way, it seems to be something we can handle later when APIs start getting added to these.

Reply:
Xamarin.Ide is an extension that gets combined with MonoDevelop (along with some other extensions) to make VSMac.

Reply:
@Therzok do you know why Xamarin.Ide needed IVT?

Reply:
Access to ILogger to implement telemetry: http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Log/ILogger.cs,10